package.json

```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "@inquirer/prompts": "7.5.1",
+    "@solana/web3.js": "^1.98.2",
```

Comment:
in the main PR description maybe add an "UPDATE" section and call out that this package may be deprecated and there are other packages involved but due to current dependencies we cannot update them all and maybe add that analyses in a new GH Issue and link to it in the Description?